test/small1/c99-tgmath2.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <tgmath.h>
+#include <complex.h>
+#include "testharness.h"
+
+typedef struct loc_t
+{
+  long nloc;
+} loc_t;
+
+
+void fun(const loc_t* loc) {
+    long l =8;
+    int n0 =(int)sqrt(l); // works
+    int n1 =(int)sqrt(loc->nloc); // fails
```

Comment:
Where is the cast to `void` here though? And what's the previously unsupported type that's being cast here then?

Reply:
It's in the result of the preprocesser when applied to the call to the type-generic sqrt. The illegal cast was double to void.